scripts/fetch-entity/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": ["./src"],
+  "compilerOptions": {
+    "strict": true
```

Comment:
We may start incrementally adding strict mode by virtue of lots of little `tsconfig.json`s scattered around the repo. 🤔 That's a problem for future us, though.